client/lib/plans/index.js

```diff
@@ -34,6 +42,14 @@ export function getFeatureByKey( feature ) {
 		: null;
 }
 
+export function getFeatureTitle( feature ) {
+	if ( ! featuresList[ feature ] || ! featuresList[ feature ].getTitle ) {
+		return null;
+	}
+
+	return featuresList[ feature ].getTitle();
+}
```

Comment:
why not use some helper functions to keep these kinds of functions smaller/cleaner?

```js
import get from 'lodash/get';
import noop from 'lodash/noop';

export function getFeatureTitle( feature ) {
	return get( featuresList, [ feature, 'getTitle' ], noop )()
}

// alternatively

export const getFeatureTitle = feature => get( featuresList, [ feature, 'getTitle' ], noop )();

// or if you don't care about the `null` being an `undefined`…
import invoke from 'lodash/invoke';

// will return `undefined` if the path on the object doesn't exist
export const getFeatureTitle = feature => invoke( featuresList, [ feature, 'getTitle' ] );
```